yarn.lock

```diff
@@ -1368,6 +1368,20 @@
     prop-types "^15.7.2"
     qs "^6.9.1"
 
+"@opentripplanner/core-utils@^1.2.1":
```

Comment:
Looks like this didn't replace the other core-utils. We may need to update something else...

Reply:
Are the deps in the other OTP-UI packages updated with that release of core-utils?

Reply:
This is so strange that yarn does this. Looks like you'll have to use this tool to correct it: https://www.npmjs.com/package/yarn-deduplicate.